app/libs/pdf_generator.rb

```diff
@@ -0,0 +1,28 @@
+require 'render_anywhere'
+
+class PdfGenerator
+  include RenderAnywhere
```

Comment:
try to avoid using such things as this

Reply:
@romaslmd it's actually ok
gem is wellknown. Other option is to use https://medium.com/evil-martians/new-feature-in-rails-5-render-views-outside-of-actions-2fc1181e86a8